src/Host/Client/Impl/Host/RHostConnector.cs

```diff
@@ -185,5 +185,13 @@ internal abstract class RHostConnector : IRHostConnector {
             var host = new RHost(name, callbacks, transport, null, cts);
             return host;
         }
+
+        public string HandleUrl(string url, CancellationToken ct) {
```

Comment:
This method doesn't belong to the RHostConnector. It is either should be implemented in derived class, or should have no dependency on the `IsRemote`
`